### PR TITLE
Exo qti import 7.x

### PR DESCRIPTION
**Pull request by david42**
To validate each qti file in the import

### CODE REVIEW

File: plugin/exo/Services/classes/QTI/OpenImport.php [hunk 1]
```diff
@@ -70,4 +75,59 @@ protected function getPrompt()
...
+                break;
+            case 'short':
+                $validated = $this->shortQtiValidate();
+                break;
+            case 'oneWord':
+                $validated = $this->oneWordValidate();
+                break;
+        }
+
+        return $validated;
```
Comment (stefk) [inline plugin/exo/Services/classes/QTI/OpenImport.php]: A `default` case is missing in the switch, or `$validated` should have an initial value.

**Comment by david42** [comment]
@stefk Yes, thanks.
I added a default value

**Comment by stefk** [comment]
LGTM

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**